spec/index.bs

```diff
@@ -6691,7 +6709,354 @@ An <dfn dfn>Extent3D</dfn> is a {{GPUExtent3D}}.
+        <td>{{GPUTextureComponentType/uint}}
+        <td>&checkmark;
+        <td>
+        <td>
+    <tr class='int'>
+        <td>{{GPUTextureFormat/rg8sint}}
+        <td>{{GPUTextureComponentType/sint}}
+        <td>&checkmark;
+        <td>
+        <td>
+    <tr class='separator'><td>32-bit formats<td  colspan=4>
+    <tr class='int'>
+        <td>{{GPUTextureFormat/r32uint}}
+        <td>{{GPUTextureComponentType/uint}}
+        <td>&checkmark;
+        <td>&checkmark;
+        <td>
+    <tr class='int'>
+        <td>{{GPUTextureFormat/r32sint}}
+        <td>{{GPUTextureComponentType/sint}}
+        <td>&checkmark;
+        <td>&checkmark;
+        <td>
+    <tr class='float'>
+        <td>{{GPUTextureFormat/r32float}}
+        <td>{{GPUTextureComponentType/float}}
+        <td>&checkmark;
+        <td>&checkmark;
+        <td><!-- Metal -->
+    <tr class='int'>
+        <td>{{GPUTextureFormat/rg16uint}}
+        <td>{{GPUTextureComponentType/uint}}
+        <td>&checkmark;
+        <td>
+        <td>
+    <tr class='int'>
+        <td>{{GPUTextureFormat/rg16sint}}
+        <td>{{GPUTextureComponentType/sint}}
+        <td>&checkmark;
+        <td>
+        <td>
+    <tr class='float'>
+        <td>{{GPUTextureFormat/rg16float}}
+        <td>{{GPUTextureComponentType/float}}
+        <td>&checkmark;
+        <td>&checkmark;
+        <td>&checkmark;
+    <tr class='norm'>
+        <td>{{GPUTextureFormat/rgba8unorm}}
+        <td>{{GPUTextureComponentType/float}}
+        <td>&checkmark;
+        <td>&checkmark;
+        <td>&checkmark;
+    <tr class='norm'>
+        <td>{{GPUTextureFormat/rgba8unorm-srgb}}
+        <td>{{GPUTextureComponentType/float}}
+        <td>&checkmark;
+        <td>
+        <td>&checkmark;
+    <tr class='norm'>
+        <td>{{GPUTextureFormat/rgba8snorm}}
+        <td>{{GPUTextureComponentType/float}}
+        <td>
+        <td>&checkmark;
+        <td>&checkmark;
+    <tr class='int'>
+        <td>{{GPUTextureFormat/rgba8uint}}
+        <td>{{GPUTextureComponentType/uint}}
+        <td>&checkmark;
+        <td>&checkmark;
+        <td>
+    <tr class='int'>
+        <td>{{GPUTextureFormat/rgba8sint}}
+        <td>{{GPUTextureComponentType/sint}}
+        <td>&checkmark;
+        <td>&checkmark;
+        <td>
+    <tr class='norm'>
+        <td>{{GPUTextureFormat/bgra8unorm}}
+        <td>{{GPUTextureComponentType/float}}
+        <td>&checkmark;
+        <td><!-- D3D12 -->
+        <td>&checkmark;
+    <tr class='norm'>
+        <td>{{GPUTextureFormat/bgra8unorm-srgb}}
+        <td>{{GPUTextureComponentType/float}}
+        <td>&checkmark;
+        <td>
+        <td>&checkmark;
+    <tr class='norm'>
+        <td>{{GPUTextureFormat/rgb10a2unorm}}
+        <td>{{GPUTextureComponentType/float}}
+        <td>&checkmark;
+        <td>
+        <td>&checkmark;
+    <tr class='float'>
+        <td>{{GPUTextureFormat/rg11b10ufloat}}
+        <td>{{GPUTextureComponentType/float}}
+        <td>&checkmark;
```

Comment:
Dawn currently has "false" here, but I'm not sure why.

Reply:
It is because Vulkan doesn't guarantee support for it.

Reply:
ah, yeah. I tried to check this but I think I read the table wrong.

Reply:
Right, I see. Fixing this.